examples/controls/control.css

```diff
@@ -0,0 +1,12 @@
+#controlDiv {
+      position:absolute;
+      top:10px;
+      right:10px;
+      width:150px;
+      height:150px;
```

Comment:
Consider using `rem` units instead of px to accomodate with different screen size and density.